### DISCUSSION TITLE
[Feature request] Sorting the homepage

### DISCUSSION THREAD

**Discussion by superm1991** [ChristopherKlay/StadiaEnhanced#102] [Ideas & Requests]
Hi,

Would it be possible to override the sorting of games in the homepage?

Specifically:
- Sort by alphabetical order (should be super easy)
- Sort by last played (current default)
- Sort by favorites*

(*) For the favorites, you could use the already working wishlist system:
- Wishlist item that is neither Claimed nor Purchased => only appears as regular wishlist item (store filter)
- Wishlist item that is Purchased => only appears as homepage favorite (sorted as to appear before the rest of the games)
- Wishlist item that is Claimed => regular wishlist + homepage favorite

What do you think?

**Comment by ChristopherKlay** [comment]
> Would it be possible to override the sorting of games in the homepage?

Absolutely. Going to add it to the to-do list on the project board.

> Sort by favorites

This would also be very do-able, but i'm not yet sure if i would want to mix the two, because having both alphabetical + wishlist ordering at the same time could definitely create some chaos.

Would a separate option to add games to your favorites, so they get displayed first in the list with a little colored border or separate grid above the current one be a solution? So they always stay at the top and simply ignore list orders.

**Reply by superm1991** [reply]
> Would a separate option to add games to your favorites, so they get displayed first in the list with a little colored border or separate grid above the current one be a solution?

Yes, of course! It's even better.
I only suggested reusing the wishlist because it was the solution with the least amount of work (or at least it seemed that way to me) :)
